Kernel/Memory/TypedMapping.h

```diff
@@ -28,6 +28,19 @@ struct TypedMapping {
     size_t length { 0 };
 };
 
+template<>
+struct TypedMapping<void> {
```

Comment:
:T If you're using `void*`, then it's not exactly a *typed* mapping, is it? Is there really no overlapping type we can use here for EHCI? If there really isn't we should just use a volatile void* to show that the mapping is super unusual/annoying.